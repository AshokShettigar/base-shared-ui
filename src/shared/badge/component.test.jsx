import { shallow } from 'enzyme';

import Badge from './component';

const defaultProps = {
    children: <div>Child</div>,
    color: 'red',
    href: 'http://test.com'
};

it('should render component', () => {
    const wrapper = shallow(<Badge {...defaultProps} />);

    expect(wrapper).toMatchSnapshot();
});
