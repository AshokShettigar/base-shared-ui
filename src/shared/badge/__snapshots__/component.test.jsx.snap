// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<span
  className="root red"
  data-component="Badge"
>
  <div>
    Child
  </div>
</span>
`;
